src/style.css

```diff
@@ -1,3 +1,84 @@
 body {
   margin: 0;
+  font-family: Roboto, Arial, sans-serif;
+  box-sizing: border-box;
+
+  --blue-color: #00acdc;
+}
+
+@font-face {
+  font-family: Roboto;
+  src: url(fonts/Roboto-Medium.ttf);
+}
+
+.wrapper {
+  margin: 0 50px;
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+}
+
+.navigation {
+  display: block;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  text-transform: uppercase;
+  background-color: white;
+  font-size: 12px;
+}
+
+.nav_list {
+  list-style: none;
+  display: flex;
+  flex-direction: row;
+  margin: 0;
+  padding: 0;
+  justify-content: center;
+}
+
+.nav_link {
+  text-decoration: none;
+  height: 60px;
+  line-height: 60px;
+  display: block;
+  color: black;
+  justify-content: center;
+  align-self: center;
+  letter-spacing: normal;
+  font-weight: 100;
+}
+
+.nav_item:nth-child(n + 2) {
+  margin-left: 20px;
+}
+
+.navigation_logo {
+  align-self: center;
+  justify-content: center;
+  height: 40px;
+  display: block;
+  float: left;
```

Comment:
Avoid using `float` for layout purposes. Modern layout techniques like flexbox and grid are more powerful and easier to maintain. In this case, you are already using flexbox for your layout, so there's no need to use `float`.